cardano-crypto-class/test/Test/Crypto/Util.hs

```diff
@@ -69,6 +79,27 @@ nullTestSeed :: TestSeed
 nullTestSeed = TestSeed (0, 0, 0, 0, 0)
 
 
+instance Arbitrary TestSeed where
+  arbitrary =
+    (\w1 w2 w3 w4 w5 -> TestSeed (w1, w2, w3, w4, w5)) <$>
```

Comment:
No love for
````
      TestSeed . (,,,,) <$> gen <*> gen <*> gen <*> gen <*> gen
````

Reply:
Addressed in https://github.com/input-output-hk/cardano-base/pull/113